src/vivarium_gates_mncnh/constants/data_values.py

```diff
@@ -351,3 +352,27 @@ class __NeonatalInterventions(NamedTuple):
 
 
 NEONATAL_INTERVENTIONS = __NeonatalInterventions()
+
+
+AZITHROMYCIN_FACILITY_TYPE_DISTRIBUTION = {
+    "Ethiopia": {
+        NO_AZITHROMYCIN_RISK.P_AZITHROMYCIN_HOME: get_norm(0.0, 0.00**2),
+        NO_AZITHROMYCIN_RISK.P_AZITHROMYCIN_BEMONC: get_norm(0.0, 0.00**2),
+        NO_AZITHROMYCIN_RISK.P_AZITHROMYCIN_CEMONC: get_norm(0.0, 0.00**2),
+    },
+    "Nigeria": {
+        NO_AZITHROMYCIN_RISK.P_AZITHROMYCIN_HOME: get_norm(0.0, 0.00**2),
+        NO_AZITHROMYCIN_RISK.P_AZITHROMYCIN_BEMONC: get_norm(0.0, 0.00**2),
+        NO_AZITHROMYCIN_RISK.P_AZITHROMYCIN_CEMONC: get_norm(0.0, 0.00**2),
+    },
+    "Pakistan": {
+        NO_AZITHROMYCIN_RISK.P_AZITHROMYCIN_HOME: get_norm(0.0, 0.00**2),
+        NO_AZITHROMYCIN_RISK.P_AZITHROMYCIN_BEMONC: get_norm(0.0, 0.00**2),
+        NO_AZITHROMYCIN_RISK.P_AZITHROMYCIN_CEMONC: get_uniform_distribution_from_limits(
+            0.153, 0.253
+        ),
+    },
+}
+AZITHROMYCIN_RELATIVE_RISK_DISTRIBUTION = get_norm(
+    1.54, 0.08**2
+)  # RR of no azithromycin intervention
```

Comment:
Hi sorry there is a late-breaking update to how to model the uncertainty here. Sorry for the change up and let me know if there are any questions! https://github.com/ihmeuw/vivarium_research/pull/1673